client/src/services/contactQuestioning.service.ts

```diff
@@ -0,0 +1,42 @@
+
+import { Subject } from 'rxjs';
+import { store } from 'redux/store';
+
+const duplicateIdentitySubject = new Subject<IdentityData[]>();
+
+const checkForDuplicates = () => {
+    const interactedContacts = store.getState().interactedContacts.interactedContacts;
+    let result = [];
+    for (var i = 0; i < interactedContacts.length; i++) {
+        let contact = interactedContacts[i];
+        if (contact.identificationType && contact.identificationNumber && contact.identificationNumber !== "") {
+            let identificationType = contact.identificationType?.id || contact.identificationType;
+            let index = interactedContacts.findIndex(obj => (obj.identificationType === identificationType || obj.identificationType?.id === identificationType) && obj.identificationNumber === contact.identificationNumber)
+            if (index !== i)
+                result.push(new IdentityData(identificationType as number, contact.identificationNumber));
+        }
+    }
+    duplicateIdentitySubject.next(result);
+
+
```

Comment:
remove

Reply:
Done

Reply:
you did not remove it

Reply:
I don't understand.. do you mean to remove the empty lines?